Sources/NIOSSH/Connection State Machine/SSHConnectionStateMachine.swift

```diff
@@ -60,11 +60,11 @@ struct SSHConnectionStateMachine {
     /// The state of this state machine.
     private var state: State
 
-    private static let defaultTransportProtectionSchemes: [NIOSSHTransportProtection.Type] = [
+    public static let bundledTransportProtectionSchemes: [NIOSSHTransportProtection.Type] = [
```

Comment:
I decided to make this public so clients can have an ability to specify a fully custom set of algorithms (i.e. exclude bundled). But if they need to add custom, they will be able to specify `bundled` + custom

Reply:
Ah, `NIOSSHConnectionStateMachine` is not public, moved to `SSHConnectionRole`